src/apps/testapps/testCellToLocalIjInternal.c

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2018-2019 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/** @file
+ * @brief tests H3 index to local IJ and IJK+ grid functions.
+ *
+ *  usage: `testCellToLocalIjInternal`
+ */
+
+#include <h3api.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "algos.h"
+#include "baseCells.h"
+#include "constants.h"
+#include "h3Index.h"
+#include "h3api.h"
+#include "localij.h"
+#include "test.h"
+#include "utility.h"
+
+SUITE(h3ToLocalIj) {
```

Comment:
Nit: Doesn't much matter, but should these have the `-Internal` suffix in the suite name?

Reply:
We can -- although maybe we should just use a macro like `__FILE__` or an extension macro like `__FILE_NAME__` instead.